src/nvim/normal.c

```diff
@@ -4652,15 +4652,18 @@ static void nv_zet(cmdarg_T *cap)
 /*
  * "Q" command.
  */
-static void nv_exmode(cmdarg_T *cap)
+static void nv_regrecorded(cmdarg_T *cap)
 {
-  /*
-   * Ignore 'Q' in Visual mode, just give a beep.
-   */
-  if (VIsual_active) {
-    vim_beep(BO_EX);
-  } else if (!checkclearop(cap->oap)) {
-    do_exmode();
+  if (checkclearop(cap->oap)) {
+    return;
+  }
+
+  while (cap->count1-- && !got_int) {
+    if (do_execreg(reg_recorded, false, false, false) == false) {
+      beep_flush();
```

Comment:
nv_at uses clearopbeep() here, is it not used here intentionally?

Reply:
I guess it's an error from me.